server/src/main/java/org/elasticsearch/cluster/routing/allocation/DataTier.java

```diff
@@ -60,7 +76,6 @@ public static boolean validTierName(String tierName) {
     /**
      * Based on the provided target tier it will return a comma separated list of preferred tiers.
      * ie. if `data_cold` is the target tier, it will return `data_cold,data_warm,data_hot`
-     * This is usually used in conjunction with {@link DataTierAllocationDecider#TIER_PREFERENCE_SETTING}
```

Comment:
Think we can keep this, the setting is still in scope here.

Reply:
++ brought this back

Reply:
u sure? I don't see it?

Reply:
Now :) Sorry accidentally put this on the top level class.